app/scripts/filters/util.js

```diff
@@ -528,6 +528,31 @@ angular.module('openshiftConsole')
       }
       var uri = new URI(url);
       var protocol = uri.protocol();
-      return uri.is('absolute') && (protocol === 'http' || protocol === 'https');      
+      return uri.is('absolute') && (protocol === 'http' || protocol === 'https');
+    };
+  })
+  // currently used to set an alt text value for an env var that has a
+  // valueFrom attribute instead of a value.
+  .filter('altTextForValueFrom', function() {
+    return function(obj) {
+      if(obj.value) {
+        return;
+      }
+      _.each(obj.valueFrom, function(val, key) {
+        console.log(key);
+        if(key === 'configMapKeyRef') {
+          obj.valueAlt = 'Set to the key ' + key + ' in config map ' + val.name;
```

Comment:
Oop, you are right. Tinkered a bit, looks like this now:

<img width="820" alt="screen shot 2016-08-11 at 5 05 18 pm" src="https://cloud.githubusercontent.com/assets/280512/17604930/ff50064a-5fe5-11e6-9cde-ee298cefc395.png">

I believe thats right/consistent with your suggestion.